<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QDialog" name="optionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Units</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="weight">
           <property name="title">
            <string>Weight</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="weight_si">
              <property name="text">
               <string>Use SI units</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="weight_us">
              <property name="text">
               <string>Use US traditional units</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="weight_imperial">
              <property name="text">
               <string>Use British imperial units</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="temperature">
           <property name="title">
            <string>Temperature</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="celsius">
              <property name="text">
               <string>Celsius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="fahrenheit">
              <property name="text">
               <string>Fahrenheit</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="volume">
           <property name="title">
            <string>Volume</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="volume_si">
              <property name="text">
               <string>Use SI units</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="volume_us">
              <property name="text">
               <string>Use US traditional units</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="volume_imperial">
              <property name="text">
               <string>Use British imperial units</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Gravity</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QRadioButton" name="radioButton_sg">
              <property name="text">
               <string>20C/20C Specific Gravity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_plato">
              <property name="text">
               <string>Plato/Brix/Balling</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Color</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QRadioButton" name="radioButton_srm">
              <property name="text">
               <string>Use SRM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_ebc">
              <property name="text">
               <string>Use EBC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Formulas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Color</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBox_mosher">
              <property name="text">
               <string>Mosher's approximation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_daniel">
              <property name="text">
               <string>Daniels' approximation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_morey">
              <property name="text">
               <string>Morey's approximation</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>IBU</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="checkBox_tinseth">
              <property name="text">
               <string>Tinseth's approximation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_rager">
              <property name="text">
               <string>Rager's approximation</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Database Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dbDir">
           <property name="toolTip">
            <string>Where your brewtarget .xml files are.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dbDirBrowse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_dbDirDefault">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>173</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_language">
      <attribute name="title">
       <string>Language</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QPushButton" name="pushButton_cs">
           <property name="toolTip">
            <string>Czech</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagCzech.svg</normaloff>:/images/flagCzech.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ca">
           <property name="toolTip">
            <string>Catalan</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagCatalonia.svg</normaloff>:/images/flagCatalonia.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_de">
           <property name="toolTip">
            <string>German</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagGermany.svg</normaloff>:/images/flagGermany.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_en">
           <property name="toolTip">
            <string>English</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagUK.svg</normaloff>:/images/flagUK.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_es">
           <property name="toolTip">
            <string>Spanish</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagSpain.svg</normaloff>:/images/flagSpain.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_fr">
           <property name="toolTip">
            <string>French</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagFrance.svg</normaloff>:/images/flagFrance.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_it">
           <property name="toolTip">
            <string>Italian</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagItaly.svg</normaloff>:/images/flagItaly.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_nl">
           <property name="toolTip">
            <string>Dutch</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagNetherlands.svg</normaloff>:/images/flagNetherlands.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pl">
           <property name="toolTip">
            <string>Polish</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagPoland.svg</normaloff>:/images/flagPoland.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pt">
           <property name="toolTip">
            <string>Portuguese</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagPortugal.svg</normaloff>:/images/flagPortugal.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ru">
           <property name="toolTip">
            <string>Russian</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../build/brewtarget.qrc">
             <normaloff>:/images/flagRussia.svg</normaloff>:/images/flagRussia.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>348</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;qt&gt;
   &lt;b&gt;Know another language?&lt;/b&gt;
   &lt;br&gt;&lt;br&gt;
   Or, would you like to improve a translation? Help us out and
  &lt;a href=&quot;https://sourceforge.net/sendmessage.php?touser=938941&quot;&gt;
  provide a translation&lt;/a&gt; so that your friends can use brewtarget!
&lt;/qt&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>weight_si</tabstop>
  <tabstop>weight_us</tabstop>
  <tabstop>weight_imperial</tabstop>
  <tabstop>celsius</tabstop>
  <tabstop>fahrenheit</tabstop>
  <tabstop>volume_si</tabstop>
  <tabstop>volume_us</tabstop>
  <tabstop>volume_imperial</tabstop>
  <tabstop>radioButton_sg</tabstop>
  <tabstop>radioButton_plato</tabstop>
  <tabstop>radioButton_srm</tabstop>
  <tabstop>radioButton_ebc</tabstop>
  <tabstop>checkBox_mosher</tabstop>
  <tabstop>checkBox_daniel</tabstop>
  <tabstop>checkBox_morey</tabstop>
  <tabstop>checkBox_tinseth</tabstop>
  <tabstop>checkBox_rager</tabstop>
  <tabstop>lineEdit_dbDir</tabstop>
  <tabstop>pushButton_dbDirBrowse</tabstop>
  <tabstop>pushButton_dbDirDefault</tabstop>
  <tabstop>pushButton_cs</tabstop>
  <tabstop>pushButton_de</tabstop>
  <tabstop>pushButton_en</tabstop>
  <tabstop>pushButton_es</tabstop>
  <tabstop>pushButton_fr</tabstop>
  <tabstop>pushButton_it</tabstop>
  <tabstop>pushButton_pl</tabstop>
  <tabstop>pushButton_pt</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../build/brewtarget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>optionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>optionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
